bin/s2nc.c

```diff
@@ -288,6 +296,8 @@ int main(int argc, char *const *argv)
     char *psk_optarg_list[S2N_MAX_PSK_LIST_LENGTH];
     size_t psk_list_len = 0;
     char *early_data = NULL;
+    uint64_t perf_send_amount = 0;
+    uint64_t perf_receive_amount = 1000000; /* 1MB */
```

Comment:
Why would this default not be 0?

Reply:
i figured the default should send at least some data. And since downloading from a server is pretty common this seemed like an appropriate default.